localstack/services/dynamodb/provider.py

```diff
@@ -729,6 +729,14 @@ def transact_write_items(
                 if inner_item:
                     existing_items.append(ItemFinder.find_existing_item(inner_item))
 
+        client_token: str | None = transact_write_items_input.get("ClientRequestToken")
+
+        if client_token:
+            # we sort the payload since identical payload but with different order could cause
+            # IdempotentParameterMismatchException error if a client token is provided
+            sorted_dict = sort_dict(json.loads(context.request.data))
+            context.request.data = json.dumps(sorted_dict).encode("utf-8")
```

Comment:
nit: we could use the `to_bytes(..)` util function here, instead of doing the `.encode(..)` call manually.